apps/asap-server/src/middleware/permission-handler.ts

```diff
@@ -0,0 +1,10 @@
+import Boom from '@hapi/boom';
+import { RequestHandler } from 'express';
+
+export const permissionHandler: RequestHandler = async (req, _res, next) => {
+  if (!req.loggedUser || req.loggedUser.onboarded !== true) {
```

Comment:
Would `loggedInUser` perhaps be clearer than `loggedUser`? As my first thought when reading this was to do with logging

Reply:
et voila https://github.com/yldio/asap-hub/pull/896/commits/80293c3f189ecaf6d35466e42380eebbab453396